src/conditionals/programs_with_decision_points/hard/q11/Discount.java

```diff
@@ -0,0 +1,34 @@
+package conditionals.programs_with_decision_points.hard.q11;
+/*
+Write a Java question that helps a store cashier determine if a given purchase qualifies for the birthday discount.
```

Comment:
Change question text to:
`Write a Java question that helps a store cashier determine if a purchase qualifies for the birthday discount. To promote customer loyalty, the store gives all its customers a 10% discount for all purchases within 45 days of their birthday. Ask the user for their birthday (day and month only) as integers and calculates if today is 45 days from that day. The program should display whether the purchase qualifies for a 10% discount or not. Assume the current date is 27/09 and all months have 30 days. If the person qualifies for the discount print out "You qualify for a 10% discount!", and if not, print "You do not qualify for any discounts.".`